vowpalwabbit/CMakeLists.txt

```diff
@@ -275,6 +278,7 @@ set(vw_all_sources
 )
 
 add_library(vw STATIC ${vw_all_sources} ${vw_all_headers})
+add_dependencies(vw fbschemas)
```

Comment:
the add_subdirectory(parser/flatbuffer) should be in this file to avoid the file above this one bringing in a dependency this file needs